transmogrifier/sources/xml/datacite.py

```diff
@@ -255,6 +255,15 @@ def get_optional_fields(self, xml: Tag) -> dict | None:
                 source_record_id,
             )
 
+        # publishers
+        if publisher := xml.metadata.find("publisher", string=True):
```

Comment:
[Non-blocking] Initially, I was wondering this would work for records where there are multiple XML elements tagged with "publisher", but I remembered that `Datacite` is a transformer used for the following sources:
* `zenodo`: Every record has a single `publisher` element ("Zenodo")
* `jpal`: Every record has a single `publisher` element ("Harvard Dataverse")
This was explored as part of[ this comment on ADR-0003](https://github.com/MITLibraries/transmogrifier/pull/132#discussion_r1507657483).

Just noting that this might be useful information to keep somewhere (maybe we can start a new Markdown file in `docs/` at some point! This Markdown file could share a link to the Datacite schema or just a description of this finding.

Reply:
I think we can have a big discussion about documentation for all the sources but I think that's beyond the scope of this PR